hoot-services/src/main/java/hoot/services/controllers/job/JobResource.java

```diff
@@ -106,4 +118,68 @@ public JobStatusResponse getJobStatus(@Context HttpServletRequest request, @Path
 
         return response;
     }
+
+    /**
+     * Deletes a job status record
+     *
+     * DELETE hoot-services/job/{id}
+     *
+     *
+     * @param jobId
+     *            ID of job record to be deleted
+     * @return id of the deleted job
+     */
+    @DELETE
+    @Path("/{jobId}")
```

Comment:
Not a fan of the path but i guess since its marked as @DELETE its clear

Reply:
this is the restful way of doing delete